ui/pages/DataManagement/DataManagement.jsx

```diff
@@ -22,6 +23,7 @@ export const DATA_MANAGEMENT_PAGES = [
   { path: 'sample_qc', component: SampleQc },
   { path: 'rna_seq', component: RnaSeq },
   { path: 'users', component: Users },
+  { path: 'lirical_exomiser', component: PhenotypePri },
```

Comment:
we do not want to hardcode lirical or expmiser anywhere

Reply:
Change it to 'phenotype_prioritization`.